packages/node/test/integration/cant-take-action.spec.ts

```diff
@@ -112,43 +112,46 @@ describe("Node method follows spec - fails with improper action taken", () => {
         }
       };
 
-      const multisigAddress = await getNewMultisig(nodeA, [
+      nodeA.on(
+        NODE_EVENTS.CREATE_CHANNEL,
+        async (data: NodeTypes.CreateChannelResult) => {
+          const tttAppInstanceProposalReq = makeTTTAppInstanceProposalReq(
+            nodeB.publicIdentifier,
+            // @ts-ignore
```

Comment:
I don't see why we need to ignore this versus type it correctly

Reply:
This can be fixed by defining a custom namespace named `NodeJs` with an interface named `Global` that has the properties we're setting/getting (i.e. `networkContext`, etc) since these properties don't exist on the pre-existing `NodeJS.Global` type.

Reply:
@cf19drofxots You can work around that using string literals like `global["networkContext"]`. That doesn't require ts-ignoring.

Reply:
Yeah doing that right now.

Reply:
I wanted to specify an interface so it would error if the string literal was typo'ed but it's giving me weird behavior that I'm not gonna spend more time trying to resolve.